polls/utils.py

```diff
@@ -32,10 +33,49 @@ def check_user_work_permission(user):
         and time_diff_minutes < minutes_after_can_continue
     )
     can_continue = time_diff_minutes > minutes_after_can_continue
-    rest_time = round(minutes_after_can_continue - time_diff_minutes) + 1
+    rest_time = round(minutes_after_can_continue - time_diff_minutes)
     return can_continue, should_rest, rest_time
 
 
+def get_user_num_tasks(user):
+    """Generate number of task a user completed then return
+    the value to be used in template tags
+    """
+    task_count = Annotation.objects.get(user=user).annotate(total_task=Count("task"))
```

Comment:
I think we should be using filter instead of "get". Because a user can have multiple annotations and this will break in that case